_component_design/pills.md

```diff
@@ -5,128 +5,244 @@ secondary: Editing
 
-{% figure [caption:"The pill pattern"] [class:"image bg-light border"] %}
-![]({{site.url}}/assets/images/components/content-areas/pills/pills-default.png){:width="499px"}
+{% figure [caption:"Pill spacing"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/content-areas/pills/pills_spacing.png){:width="418px"}
 {% endfigure %}
 
 {% endcolumn %}
 
 </div>
-  
-</section> 
 
-## Specifications
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+
+## States
+
+Pills can exist in active, hover, too many, or too long states.
+
+{% endcolumn %}
+
+</div>
+
+</section>
 
 <section class="static-section" markdown="1">
 
 <div class="hxRow" markdown="1">
 
 {% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
-### Style
+### Active
+The active state is the default state, and indicates that a user has made a selection.
 
-While pills may be larger or smaller than body font, generally use body font.
+{% endcolumn %}
 
-Pills consist of a container, text, and an “x” icon to indicate removal. By default, they are `gray 400` background.
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
 
-### Spacing
+{% figure [caption:"Active state"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/content-areas/pills/pills_states_active.png){:width="464px"}
+{% endfigure %}
+
+{% endcolumn %}
 
-Pills have 16px of space on either side and 8px below and beside when grouped together. Put pills 4px beneath other associated components&mdash;use the best top spacing for your particular use case. Unless there is a legitimate reason, justify pills to the left.
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
 
-If more than 15 pills are bunched together, consider an alternative pattern or contact the Helix team.
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Hover
+The hover state indicates that the user can remove the selection.
 
 {% endcolumn %}
 
 {% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
 
-{% figure [caption:"The pill pattern"] [class:"image bg-light border"] %}
-![]({{site.url}}/assets/images/components/content-areas/pills/pills-specs.png){:width="509px"}
+{% figure [caption:"Hover state"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/content-areas/pills/pills_states_hover.png){:width="463px"}
 {% endfigure %}
 
 {% endcolumn %}
 
 </div>
-  
-</section> 
+
+</section>
 
 <section class="static-section" markdown="1">
 
 <div class="hxRow" markdown="1">
 
 {% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
-### States
+### Too many
+Use the too many state when there is a group of more than 15 pills.
```

Comment:
this seems arbitrary. esp since the horizontal size can vary. can we rephrase for more clarity in what to do when there is vertical or horizontal space constraints?